book/src/concepts/register_types.md

```diff
@@ -23,15 +23,11 @@ Then you register the QML Type in the normal way.
 {{#include ../../../examples/qml_minimal/cpp/main.cpp:book_qml_register}}
 ```
 
-Note in the future there may be a helper to call which could register all the types even when not using a plugin ( [https://github.com/KDAB/cxx-qt/issues/33](https://github.com/KDAB/cxx-qt/issues/33) ).
+## Using a plugin
 
-## Using QQmlExtensionPlugin
+If you are [creating a C++ plugin for QML](https://doc.qt.io/qt-6/qtqml-modules-cppplugins.html) then ensure any of the normal changes to a CMake projects to build the plugin as a library, add the `qmldir`, have a C++ file which derives from `QQmlExtensionPlugin` and registers the types, and ensure that the generated library is in the import path.
```

Comment:
```suggestion
If you are [creating a plugin for QML](https://doc.qt.io/qt-6/qtqml-modules-cppplugins.html) then ensure any of the normal changes to a CMake projects to build the plugin as a library, add the `qmldir`, have a C++ file which derives from `QQmlExtensionPlugin` and registers the types, and ensure that the generated library is in the import path.
```

Mentioning `C++` there is confusing because the application logic is in Rust.

Reply:
Done